pkg/controllers/updaterun/controller.go

```diff
@@ -185,11 +185,19 @@ func (r *Reconciler) ensureFinalizer(ctx context.Context, updateRun *placementv1
 
 // recordUpdateRunSucceeded records the succeeded condition in the ClusterStagedUpdateRun status.
 func (r *Reconciler) recordUpdateRunSucceeded(ctx context.Context, updateRun *placementv1beta1.ClusterStagedUpdateRun) error {
+	meta.SetStatusCondition(&updateRun.Status.Conditions, metav1.Condition{
+		Type:               string(placementv1beta1.StagedUpdateRunConditionProgressing),
+		Status:             metav1.ConditionFalse,
+		ObservedGeneration: updateRun.Generation,
+		Reason:             condition.UpdateRunSucceededReason,
+		Message:            "All stages are completed",
+	})
 	meta.SetStatusCondition(&updateRun.Status.Conditions, metav1.Condition{
 		Type:               string(placementv1beta1.StagedUpdateRunConditionSucceeded),
 		Status:             metav1.ConditionTrue,
 		ObservedGeneration: updateRun.Generation,
 		Reason:             condition.UpdateRunSucceededReason,
+		Message:            "ClusterStagedUpdateRun completed successfully",
```

Comment:
```suggestion
		Message:            "All stages are completed successfully",
```
use consistent terminology?